src/features/calendar/components/CalendarWeekView/index.tsx

```diff
@@ -1,5 +1,88 @@
-const CalendarWeekView = () => {
-  return <h2>Week</h2>;
+import { Box } from '@mui/system';
+import dayjs from 'dayjs';
+import { FormattedTime } from 'react-intl';
+import isoWeek from 'dayjs/plugin/isoWeek';
+import { Typography } from '@mui/material';
+
+import DayHeader from './DayHeader';
+import range from 'utils/range';
+import theme from 'theme';
+
+dayjs.extend(isoWeek);
+
+const HOUR_HEIGHT = 5;
+
+export interface CalendarWeekViewProps {
+  focusDate: Date;
+}
+
+const CalendarWeekView = ({ focusDate }: CalendarWeekViewProps) => {
+  const focusWeekStartDay =
+    dayjs(focusDate).isoWeekday() == 7
+      ? dayjs(focusDate).add(-1, 'day')
+      : dayjs(focusDate);
+
+  return (
+    <Box
+      columnGap={1}
+      display="grid"
+      gridTemplateColumns={'auto repeat(7, 1fr)'}
+      gridTemplateRows={'auto 1fr'}
+      rowGap={1}
+    >
+      {/* Week Number */}
+      <Box alignItems="center" display="flex">
+        <Typography color={theme.palette.grey[500]} variant="caption">
+          {focusWeekStartDay.isoWeek()}
+        </Typography>
+      </Box>
+      {/* Day headers across the top */}
+      {range(7).map((weekday: number) => {
+        const weekdayDate = focusWeekStartDay.day(weekday + 1);
+        return (
+          <DayHeader
+            key={weekday}
+            date={weekdayDate}
+            focused={
+              new Date().toDateString() == weekdayDate.toDate().toDateString()
+            }
+          />
+        );
+      })}
+      {/* Hours column */}
```

Comment:
I think this needs to go back into a container in order for the sticky header while scrolling to work.

Reply:
Yep, done